ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,942 @@
+include::../shared-doc/attributes.adoc[]
+
+= ejb-txn-remote-call: Demonstrates remote EJB calls and transaction propagation
+:author: Ondra Chaloupka
+:level: Intermediate
+:technologies: EJB, JTA, Clustering
+
+[abstract]
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productName}.
+
+:standalone-server-type: ha
+:archiveType: war
+:uses-h2:
+:requires-multiple-servers:
+:jbds-not-supported:
+
+== What is it?
+
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productNameFull}.
+
+== Description
+
+The EJB remote calls contains a JTA transaction which is propagated over the EJB call.
```

Comment:
"The EJB remote calls contains "

Subject-verb agreement. 

Is this specifically describing the calls  in this quickstart?  Or calls in general.  I recommend clarifying either way.

"contains a JTA transaction which is propagated"

"which" > "that"

We follow the IBM Style Guide, which calls for American usage of relative pronouns.  American usage is more strict in regard to usage of "that" and "which".

Reply:
I changed the "which" to "that".
To your first question - I meant the calls in this quickstart.

Reply:
Did you edit to clarify?  For some reason, I am not seeing revisions.  Perhaps you have not committed and pushed them yet.

Reply:
Yes, I added the comment before I've really pushed the update to github. I assume I should rather used the review for this. Now it should be updated.